.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,52 @@
+name: Renovate
+on:
+  push:
+    branches:
+      - release-*
+
+jobs:
+  renovate:
+    name: Renovate
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Install node
+        uses: actions/setup-node@v4
+      - name: Install json5
+        run: npm install json5
```

Comment:
Why not do it in Python? (we already have Python scripts)

https://pypi.org/project/json5/

Reply:
+1, not a fan of Javascript

Reply:
It's done in python now 🥳